docs/lib/analytics/fragments/android/escapehatch.md

```diff
@@ -0,0 +1,23 @@
+For advanced use cases where Amplify does not provide the functionality, you can retrieve the escape hatch to access AWSPinpoint instance.
+
+<amplify-block-switcher>
+<amplify-block name="Java">
+
+```java
+AWSPinpointAnalyticsPlugin plugin = (AWSPinpointAnalyticsPlugin) Amplify
+        .Analytics
+        .getPlugin("amazonPinpointAnalyticsPlugin");
```

Comment:
is this correct on android? i think it should be `awsPinpointAnalyticsPlugin`

Reply:
I just moved this I didnt change it?

Reply:
fixed this